Bottom: d5820dd1b6d9da43d13b33e6b6ed4d395225239c
Top:    c0d14cdb6d211a64403d4761d1d1b53b6a488393
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-26 23:37:04 +0100

Refresh of rename-component-solder

---

diff --git a/src/move.c b/src/move.c
index 42b077c..c9e8628 100644
--- a/src/move.c
+++ b/src/move.c
@@ -921,7 +921,7 @@ LastLayerInTopGroup (int layer)
 static int
 LastLayerInBottomGroup (int layer)
 {
-  int bottom_group = GetLayerGroupNumberByNumber(bottomsilk_layer);
+  int bottom_group = GetLayerGroupNumberByNumber(bottom_silk_layer);
   int lgroup = GetLayerGroupNumberByNumber(layer);
   if (bottom_group == lgroup
       && PCB->LayerGroups.Number[lgroup] == 2)
